model_analyzer/reports/report_manager.py

```diff
@@ -857,10 +858,9 @@ def _build_detailed_info(self, model_config_name):
         gpu_cpu_string = "CPU"
 
         if not run_config.cpu_only():
-            gpu_dict = self._get_gpu_stats(measurements=measurements)
-            gpu_names = ','.join(list(gpu_dict.keys()))
-            max_memories = ','.join([str(x) + ' GB' for x in gpu_dict.values()])
-            gpu_cpu_string = f"GPU(s) {gpu_names} with memory limit(s) {max_memories}"
+            (gpu_names,
+             max_memories) = self._get_gpu_stats(measurements=measurements)
```

Comment:
nit: I think we can make it a single line for better readability: 
`gpu_names, max_memories = self._get_gpu_stats(measurements)`